packages/manager/apps/dedicated/client/app/components/services/services.constants.js

```diff
@@ -1,138 +1,140 @@
-angular.module('App').constant('SERVICES_TARGET_URLS', {
+export const SERVICES_TARGET_URLS = {
```

Comment:
Depending on the use we might be able to replace those with the configuration exposed in a 2APU

Reply:
`SERVICES_TARGET_URLS` is only used in `packages/manager/apps/dedicated/client/app/components/services/services.service.js` and we dont update the key, but yes, the association between api routes/universe should be updated.

In theses PR I don't define specific app and use existing and prodded apps only (`dedicated`/`public-cloud`/`web`...)